ambry-router/src/main/java/com/github/ambry/router/NonBlockingRouterMetrics.java

```diff
@@ -215,6 +215,13 @@ public class NonBlockingRouterMetrics {
   public final Counter compositeBlobGetCount;
   public final Counter rawBlobGetCount;
 
+  // Metrics for Reserving partitions and blobid for metadata chunk.
+  public final Counter numFailedPartitionReserveAttempts;
+  public final Counter unknownReservedPartitionClassCount;
```

Comment:
Shall we use the same name convention and call it numUnknownReservedPartitionClassCount?

Reply:
done.